### PR TITLE
brew: add Korean translation

### PR DESCRIPTION
**Pull request by jyoo0515**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by navarroaxel** [comment]
Hi @jyoo0515, welcome and thanks for your submission. Could you please sign the license to pass the CLA check?

**Comment by jyoo0515** [comment]
Done! Thanks for the reminder

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages.ko/common/brew.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+- 설치된 모든 공식(formulae)과 캐스크(casks) 나열:
+
+`brew list`
+
+- 설치된 특정 공식(formula) 혹은 캐스크(cask) 업그레이드 (옵션이 주어지지 않으면 모든 공식과 캐스크 업그레이드):
+
+`brew upgrade {{formula}}`
+
+- Homebrew 저장소에서 Homebrew와 함께 모든 공식(formulae)과 캐스크(casks)의 최신 버전 가져오기
```
Comment (navarroaxel) [inline pages.ko/common/brew.md:18]:
```suggestion
- Homebrew 저장소에서 Homebrew와 함께 모든 공식(formulae)과 캐스크(casks)의 최신 버전 가져오기:
```

File: pages.ko/common/brew.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+`brew update`
+
+- 최신 버전이 아닌 공식(formulae)과 캐스크(casks) 나열
```
Comment (navarroaxel) [inline pages.ko/common/brew.md:22]:
```suggestion
- 최신 버전이 아닌 공식(formulae)과 캐스크(casks) 나열:
```

File: pages.ko/common/brew.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+`brew outdated`
+
+- 사용 가능한 공식(formulae)과 캐스크(casks) 검색
+
+`brew search {{text}}`
+
+- 공식(formula) 혹은 캐스크(cask)에 대한 정보 표시 (버전, 설치 경로, 의존성 등)
```
Comment (navarroaxel) [inline pages.ko/common/brew.md:30]:
```suggestion
- 공식(formula) 혹은 캐스크(cask)에 대한 정보 표시 (버전, 설치 경로, 의존성 등):
```

File: pages.ko/common/brew.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+- Homebrew 저장소에서 Homebrew와 함께 모든 공식(formulae)과 캐스크(casks)의 최신 버전 가져오기
+
+`brew update`
+
+- 최신 버전이 아닌 공식(formulae)과 캐스크(casks) 나열
+
+`brew outdated`
+
+- 사용 가능한 공식(formulae)과 캐스크(casks) 검색
```
Comment (navarroaxel) [inline pages.ko/common/brew.md:26]:
```suggestion
- 사용 가능한 공식(formulae)과 캐스크(casks) 검색:
```

File: pages.ko/common/brew.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+`brew search {{text}}`
+
+- 공식(formula) 혹은 캐스크(cask)에 대한 정보 표시 (버전, 설치 경로, 의존성 등)
+
+`brew info {{formula}}`
+
+- 설치된 Homebrew에 문제가 있는 확인
```
Comment (navarroaxel) [inline pages.ko/common/brew.md:34]:
```suggestion
- 설치된 Homebrew에 문제가 있는 확인:
```

**Review by navarroaxel — APPROVED** [review]

**Comment by navarroaxel** [comment]
Remember that you can use this GitHub feature to apply all the suggestions in 1 commit <https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request>.

**Pull request merged**